server/app/views/admin/questions/QuestionsListView.java

```diff
@@ -461,12 +475,21 @@ abstract static class GroupedReferencingPrograms {
 
     abstract ImmutableList<ProgramDefinition> removedPrograms();
 
+    /**
+     * Returns an immutable list of program definitions that are currently in draft status with
+     * disabled visibility.
+     */
+    abstract ImmutableList<ProgramDefinition> disabledPrograms();
+
     static Builder builder() {
       return new AutoValue_QuestionsListView_GroupedReferencingPrograms.Builder();
     }
 
     boolean isEmpty() {
-      return usedPrograms().isEmpty() && addedPrograms().isEmpty() && removedPrograms().isEmpty();
+      return usedPrograms().isEmpty()
+          && addedPrograms().isEmpty()
+          && removedPrograms().isEmpty()
+          && disabledPrograms().isEmpty();
```

Comment:
suggestion (maybe blocking): I think you want to wrap this `isEmpty` check in a feature flag check as well so that the overall `isEmpty` check isn't affected.

Reply:
I don't think adding a feature flag check here is necessary. With the feature flag default being disabled, `disabledPrograms` is always empty. 
It won't affect isEmpty() since the logic under this function is to check `usedPrograms`, `addedPrograms` and `removedPrograms`.

But happy to discuss further if you think otherwise!

Reply:
I'm thinking of the situation where we're testing out the feature. Say a CE turns it on in production sets some programs to "Disabled", but then we realize there's a bug and they switch it off again. If we don't guard this check, then any programs that were set to Disabled while the flag was enabled will be counted and this check would return `false` when it should return `true`.

This check is used on line 429:
```
if (groupedReferencingPrograms.isEmpty()) {
      tag.with(p("Used in 0 programs."));
    }
```

It's also used on line 570 (although I'm not sure what the "referencing modal" is actually for):
```
    if (referencingPrograms.isEmpty()) {
      return Optional.empty();
    }
```

My brain is feeling noodley trying to parse these sets though, so let me know if I'm confused about how this is working.

Reply:
Say a CE turns it on in production sets some programs to "Disabled", but then we realize there's a bug and they switch it off again.  - I think that's a valid edge case. 

Wrapped isEmpty() with the feature flag. Thanks!

To answer the question, makeReferencingProgramsModal is the modal that gets generated once the user clicks on the phrase `added to # programs`.

Reply:
Thank you for explaining!